docs/notebooks/plot_comparisons.ipynb

```diff
@@ -0,0 +1,1958 @@
+{
```

Comment:
Yes, this can be achieved by passing 

```python3
contrast={"persons": [1, 2, 4]}
```

and since we use `list(itertools.combinations(contrast.values, 2))` where `contrast.values = [1, 2, 4]`, the contrasts end up being a pairwise combination:

`(1, 2), (1, 4), (2, 4)`

Reply:
<p>That will generate all combinations, I am asking if it is possible (or even desireable) to write something like this (not necessarily this) <code>contrast={"persons": [1, (2, 4)]}</code>  and get <code>[(1,2), (1,4)]</code> .</p><p> </p><p> </p><p> </p>